test/src/drag_select_grid_view/drag_select_grid_view_test.dart

```diff
@@ -1039,7 +1021,7 @@ void main() {
         await longPressDownAndDrag(
           tester: tester,
           finder: gridFinder,
-          offset: Offset(0, -(dragSelectState.context.size.height / 2) + 1),
+          offset: Offset(0, -(dragSelectState.context.size!.height / 2) + 1),
```

Comment:
We might want to check if `dragSelectState.context.size` can really never be `null` at this point.